src/components/CreateTweet/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import TextAreaTweet from './TextAreaTweet';
+
+import styles from './styles.module.scss';
+
+const CreateTweet = () => {
```

Comment:
Старайся компоненты называть существительными.

```suggestion
const NewTweetForm = () => {
```